thunder/tests/litgpt_model.py

```diff
@@ -118,47 +118,8 @@
 name_to_config = {config["name"]: config for config in configs}
 
 
-class OverridenKVCache(nn.Module):
```

Comment:
It's not clear why we're removing a test here... is there some reason this code is fundamentally invalid? Why wouldn't we want to test it?

Reply:
The test is there, this KVCache was used to avoid missing ops in Thunder. See `test_litgpt_variants_kvcache`

Reply:
Ahh, I understand now. Thank you!